.gitignore

```diff
@@ -88,6 +88,9 @@ docs
 cypress/videos
 cypress/screenshots
 
+# Applitools
+secrets.applitools.json
```

Comment:
Dunno if this is how we want to do this, or with shell variables, but we should discuss.

Reply:
We tend to use `.env` files for secrets. 

There's not "one" file, but one for each package that needs one.

Adding to the root env file would be fine, I think thats where it used to be